plugins/filter/filters.py

```diff
@@ -360,6 +361,10 @@ def c3_connect_properties(self, connect_group_list, groups, hostvars, ssl_enable
                     final_dict['confluent.controlcenter.connect.' + group_id + '.ssl.keystore.password'] = keystore_storepass
                     final_dict['confluent.controlcenter.connect.' + group_id + '.ssl.key.password'] = keystore_keypass
 
+                if delegate_host.get('kafka_connect_oauth_enabled', oauth_enabled) and not rbac_enabled:
+                    final_dict['confluent.controlcenter.connect.' + group_id + '.oauthbearer.login.client.id'] = oauth_user
```

Comment:
i think the client.id  property for other components is getting set from variables/vars/main.yml. So is there some specific reason for doing it in this file for c3

Reply:
these are the clients for other cp components, not kafka. C3 acts as client for connect here for which we have this filter, similar to client_properties filter for kafka clients